src/main/java/org/apache/sling/distribution/servlet/DistributionPackageImporterServlet.java

```diff
@@ -126,8 +127,10 @@ protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse
             ServletJsonUtils.writeJson(response, SC_OK, "package imported successfully", null);
 
         } catch (final Throwable e) {
-            ServletJsonUtils.writeJson(response, SC_INTERNAL_SERVER_ERROR, "an unexpected error has occurred during distribution import", null);
-            log.error("Error during distribution import", e);
+            String msg = format("an unexpected error has occurred during distribution import. Error:%s",
+                    e.getMessage());
+            log.error(msg, e);
+            ServletJsonUtils.writeJson(response, SC_INTERNAL_SERVER_ERROR, msg, null);
```

Comment:
I think that a statuscode 500 is wrong here, because it's not an internal server error. It's a situation which is handled by this code, therefor controlled. 
As above: Can we make the exception more narrow and just capture a specific type, and therefor find a better statuscode for it? Same for the exception message.

Reply:
I think currently we have no means of determining if the error is because of bad data sent by the client or because of some internal problem. 
So in my opinion 500 is ok. I also think current client would not be prepared to distinguish different status codes.
Do you have a concrete proposal how to improve the code?